xlive/H2MOD/Modules/Camera/FOV/FOV.cpp

```diff
@@ -0,0 +1,25 @@
+#include "H2MOD.h"
+#include "../Camera.h"
+
+#define _USE_MATH_DEFINES
+#include "math.h"
+
+void Camera::Tweaks::SetFOV(float field_of_view_degrees) 
+{
+	if (h2mod->Server)
+		return;
+
+	if (field_of_view_degrees > 0.f && field_of_view_degrees <= 110.f)
+	{
+		float current_FOV = *reinterpret_cast<float*>(h2mod->GetBase() + 0x41D984);
+
+		//int res_width = *(int*)(H2BaseAddr + 0xA3DA00); //wip
```

Comment:
if this code isn't ready, is this method ready for commit?

if you have stuff not ready, it should be prefixed with a TODO/issue/feature number